src/arrays.js

```diff
@@ -1,62 +1,87 @@
 const getNthElement = (index, array) => {
-  // your code here
+  console.log(index % array.length);
+  return array[index % array.length];
 };
+//copied and pasted, couldn't work out alternative.
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
-};
+  const biggerArray = array.concat(element);
+  return biggerArray;
+};
+// {
+//   const biggerArray = array.slice()
+//   biggerArray.push(element);
+//   return biggerArray;
+// };
+//I took ages on this one, and came up with a longer solution before switching to concat - much simpler.
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index,1);
 };
 
-const numbersToStrings = numbers => {
-  // your code here
+function numbersToStrings(numbers) {
+  return numbers.toString().split(',');
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
-};
+  const upperCaseStrings = strings.map(a => a.toUpperCase());
+  return upperCaseStrings;
+};
+function reverseWordsInArray(strings) {
+  return strings.map(a => {
+    return a
+      .split('')
+      .reverse()
+      .join('');
+  });
+}
 
-const reverseWordsInArray = strings => {
-  // your code here
-};
-
-const onlyEven = numbers => {
-  // your code here
-};
+function onlyEven(numbers) {
+  return numbers.filter(a => {
+    return a % 2 === 0;
+  });
+}
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+ return array.slice(0,index).concat(array.slice(index + 1));
```

Comment:
Minor change but the indentation should be 2 spaces here:
```suggestion
  return array.slice(0,index).concat(array.slice(index + 1));
```